app/models/need.rb

```diff
@@ -190,4 +196,13 @@ def self.sort_created_and_start_date(first, second)
     first.created_at <=> second.created_at
   end
   # rubocop:enable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
+  def valid_start_on?
```

Comment:
FYI we could probably use the datetime validation gem Riccardo added. Of course this is fine too